strangle.py

```diff
@@ -27,21 +27,26 @@ def _var(self, name):
             name (str): name of the variable.
 
         Returns:
-            str. the asked for string or an empty one.
+            dict. the locals containing the name or an empty one.
         """
         frame = inspect.stack()[1][0]
         while name not in frame.f_locals:
             frame = frame.f_back
             if frame is None:
-                return str()
+                return dict()
 
-        return str(frame.f_locals[name])
+        return frame.f_locals
 
     def __str__(self):
         strangled = self.origin
         for match in self.INDICATOR_PATTERN.findall(strangled):
-            indicator = match[1:-1]
+            full_indicator = match[1:-1]

```

Comment:
full_indicator and main_indicator are not very indicative names.
how about strangle_expression instead of full_indicator and i'm not entirely sure what you meant by main_indicator